cmd/mothership/mothership/start/auditlog_test.go

```
@@ -0,0 +1,179 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/kyma-incubator/reconciler/internal/cli"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	postKey       = "runtimeID"
+	postValue     = "bb7fb804-ade5-42bc-a740-3c2861d0391d"
+	tenantID      = "5f6b71a9-cd48-448d-9b58-9895f1639bc6"
+	jwtPayloadSub = "test2@test.pl"
+)
+
+type MemorySink struct {
+	*bytes.Buffer
+}
+
+func (s *MemorySink) Close() error { return nil }
+func (s *MemorySink) Sync() error  { return nil }
+
+func testLoggerWithOutput() (*zap.Logger, *MemorySink) {
+	sink := &MemorySink{&bytes.Buffer{}}
+	_ = zap.RegisterSink("memory", func(*url.URL) (zap.Sink, error) {
+		return sink, nil
+	})
+	cfg := zap.Config{
+		Encoding:         "json",
+		Level:            zap.NewAtomicLevelAt(zapcore.DebugLevel),
+		OutputPaths:      []string{"memory://"},
+		ErrorOutputPaths: []string{"memory://"},
+		EncoderConfig: zapcore.EncoderConfig{
+			MessageKey:   "",
+			LevelKey:     "level",
+			EncodeLevel:  zapcore.CapitalLevelEncoder,
+			TimeKey:      "time",
+			EncodeTime:   zapcore.ISO8601TimeEncoder,
+			EncodeCaller: zapcore.ShortCallerEncoder,
+		},
+	}
+	logger, _ := cfg.Build()
```

Don't swallow the error. Use `require.NoError(...)`